webrtc.html

```diff
@@ -12001,8 +12001,8 @@ <h3 id="sec-constraints">Constraints</h3>
       default values for attributes that would be left uninitialized without
       the init dictionary present.</li>
       <li>InvalidMediaStreamTrackError changed to InvalidParameter.</li>
-      <li>Fire NetworkError when the data transport is closed with an
-      error.</li>
+      <li>Fire the <code>RTCDataChannel.onerror</code> EventHandler
```

Comment:
Right.  Fixed the language but there is still the question about whether it is the RTCError event that should be fired, or Error.